packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -221,20 +222,20 @@ $ms-Persona-presenceSizeXl: 28px;
 }
 
 .ms-Persona-details {
+  box-sizing: border-box;
   padding: 0 12px;
   vertical-align: middle;
   @include text-align(left);
   @include padding-left($ms-Persona-sizeMd + $ms-Persona-imageDetailsLgSpace);
   display: table-cell;
+  width: 100%;
```

Comment:
We get this issue that causes the details to shift to the right and not fill the whole space if we remove width: 100%.

<img width="1071" alt="screen shot 2017-01-03 at 1 58 23 pm" src="https://cloud.githubusercontent.com/assets/1291968/21624672/cddc1a06-d1bc-11e6-9bd2-669a836e4ad6.png">